docs/configure-gcp.md

```diff
@@ -56,12 +56,28 @@ This is a setup in the account in which the service shall deploy the actual inst
 2. Navigate to IAM & Admin on GCP.
 3. Click IAM.
 4. Click +ADD.
-5. Fill in the Service Account e-mail under New principals.
-6. Choose the permissions the provisioning app needs (TBC): 
-Roles:
-   - Service Account User
-   - Compute Admin (We are allowing here more permissions than needed)
-7. Click SAVE.
+5. Fill in the Provisioning Service Account principal under New principals.
+6. Create a custom role with the permissions the provisioning app needs (TBC): 
+  a. Navigate to Roles 
+  b. Click + CREATE ROLE
+  c. Choose these permissions: 
+     - iam.serviceAccounts.actAs
+     - compute.instances.create
+     - compute.disks.create
+     - compute.subnetworks.use
+     - compute.instances.setMetadata
+     - compute.instances.setLabels
+     - compute.instances.list
+     - compute.instanceTemplates.list
+     - compute.instanceTemplates.create
+     - iam.serviceAccounts.getIamPolicy
+     - compute.instances.setServiceAccount
+     - compute.instanceTemplates.useReadOnly
+     - compute.subnetworks.useExternalIp
+     - iam.roles.get
+     - serviceusage.services.use
```

Comment:
TBC stands for To Bo Continued in this case?

For every TODO in the codebase I suggest a ticket in our Jira. :-)